src/links.js

```diff
@@ -66,7 +67,8 @@ function linkToObject(link){
   return {
     'id': link.id,
     'label': getNodeLabel(link),
-    'note': null,
+    'deleted': link.classList[link.classList.length-1] == "deleted",
```

Comment:
This line would break if we were to add more classes to the link, and might interact poorly with the "selected" class.  Could you instead use link.classList.contains("deleted")?